### DISCUSSION TITLE
Ideas for development of notifications feature

### DISCUSSION THREAD

**Discussion by srchild** [mcguirepr89/BirdNET-Pi#299] [Ideas]
The Apprise notification for new species detected could be very useful to highlight the presence of an unexpected or infrequent visitor. But I might miss my one chance to see the bird that time, and would like to be notified when it reappears so that I can be ready with my binoculars to go out and have another look for it.

Could there be more fine-grained options between "new species only" (gives only one chance to go and see it) and "all detections" (too many messages, too much noise, I would get hundreds of blackbird notifications alone during some hours!)

For example:

 - "infrequent visitor detected" (with user-specified threshold, such as fewer than five visits total, or fewer than 10 visits per week)
 - perhaps a "species of interest" list, a user-selected list of species to trigger extra notifications
 - perhaps both of the above combined, a species of interest list which can be populated manually for specific birds of interest but which is also auto-populated with birds detected with low frequency 

Thanks

**Comment by ehpersonal38** [comment]
Hi @srchild,

You'll be happy to know I've implemented your first idea "infrequent visitor detected" into main 49a92bae6cc2c8b4912914562284176568ce0501. You can update and try it out now.

Hopefully we can chip away at the rest of your ideas with time.

😄

**Reply by srchild** [reply]
Excellent! Thank you.
